mkdocs.yml

```diff
@@ -22,22 +22,35 @@ nav:
           - Creating your first Trigger: getting-started/first-trigger.md
         - What's Next?: getting-started/next-steps.md
     - Installing Knative:
```

Comment:
Maybe move this straight into an admin guide?

Reply:
The Eventing Getting Started? I view this as a developer persona thing

Reply:
Nah the install docs :)

Reply:
yes that is 100% admin guide material